ProcessMaker/Http/Controllers/Api/UserController.php

```diff
@@ -293,6 +293,11 @@ public function update(User $user, Request $request)
 
         $request->validate(User::rules($user));
         $fields = $request->json()->all();
+        if (($fields['username'] !== $user->getAttribute('username') || isset($fields['password'])) && 
```

Comment:
Please update this if to
`if ( (in_array('password' $fields) || in_array('username' $fields)) && `
but I think so the best way is to add a custom middleware
https://laravel.com/docs/10.x/middleware

Reply:
in the first i can change the password, The username will always be in fields, so I check if it is different from the original. 
I am going to study the middleware, in any case I would ask that the estimated time in the ticket be increased

Reply:
May be you can use Laravel's error-safe Arr::get ( https://laravel.com/docs/8.x/helpers#method-array-get)